src/rules/function-url-no-scheme-relative/README.md

```diff
@@ -0,0 +1,43 @@
+# function-url-no-scheme-relative
+
+Disallow scheme relative urls.
```

Comment:
I think that most people might not know what this means, so we should provide a link about it.